src/search/pruning_method.cc

```diff
@@ -13,6 +16,31 @@ PruningMethod::PruningMethod()
 void PruningMethod::initialize(const shared_ptr<AbstractTask> &task_) {
     assert(!task);
     task = task_;
+    num_unpruned_successors_generated = 0;
+    num_pruned_successors_generated = 0;
+}
+
+void PruningMethod::prune_op_ids(
+    const State &state, vector<OperatorID> &op_ids) {
+    assert(!task_properties::is_goal_state(TaskProxy(*task), state));
```

Comment:
It is not clear to me why we should not call this method in goal states, so maybe a comment would be good here. It's not necessary though (and I like it a lot that we don't delegate this to the derived classes anymore! :))

Reply:
To be honest, I don't know. Maybe something specific to stubborn sets? Maybe because the list of operators are empty for goal states? Although I don't see why this should cause problems.

Reply:
Maybe it's also that we in general shouldn't do anything with goal states. But as I said, an explanatory comment isn't really necessary anyway, so I'll just mark this as resolved.